### DISCUSSION TITLE
How can I run the test？

### DISCUSSION THREAD

**Discussion by popyy-hpo** [eclipse-theia/theia-cloud#284] [Q&A]
Hello!
When running tests for the Theia Cloud, I noticed that only the service unit test ApplicationPropertiesTests passes. The other tests, which I believe are integration tests, are not running successfully. I saw in the documentation that unit tests need to be executed in Eclipse. In a CI/CD environment without Eclipse, how can tests be run? Could you provide some suggestions? Thank you very much.

**Comment by jfaltermeier** [comment]
Hi, the Github workflow running the tests can be found here: https://github.com/eclipsesource/theia-cloud/blob/main/.github/workflows/test-service.yml

The required commands should be
```
mvn clean install -f java/common/maven-conf --no-transfer-progress
mvn clean install -f java/common/org.eclipse.theia.cloud.common --no-transfer-progress
mvn clean verify -fae -f java/service/org.eclipse.theia.cloud.service --no-transfer-progress
```
